config/features.yml

```diff
@@ -1569,6 +1570,10 @@ features:
     actor_type: user
     description: >
       When enabled, 21P-527EZ will display additional explanations for the income and assets requirement.
+  pension_medical_evidence_clarification:
+    actor_type: user
+    description: >
+      [Front-end only] When enabled, 21P-527EZ will display additional explanations for the medical evidence requirement.
```

Comment:
This looks unrelated to the change above - how did this get added, and how did some others get removed?